packages/admin/admin-icons/icons/youtube.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd">
```

Comment:
The viewbox/width seems off... shouldn't this be 16x16?

Reply:
when the viewbox is set to 16x16 the icon is distorted or cut of

Reply:
@jamesricky is this okay?

Reply:
We already have these icons
<img width="456" alt="image" src="https://github.com/user-attachments/assets/9381758f-ba00-4340-9373-75cfec0952ce">

- https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-icons/src/YouTube.tsx
- https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-icons/src/Vimeo.tsx

Reply:
@thomasdax98 mentioned that these should be removed in favor of a non tinted version

Reply:
> @thomasdax98 mentioned that these should be removed in favor of a non tinted version

This would be a breaking change, as the tinted version might be used in some projects.

Reply:
I just searched through all our projects, none of them use either icon. Should we still move the removal of the tints to next?